openslides/motions/static/js/motions/site.js

```diff
@@ -515,10 +528,35 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
                 });
         };
 
+        // Export the given motions as a csv file
+        $scope.csv_export = function (motions) {
```

Comment:
Ok. We can leave the logic there for the moment. I would say we do not need two buttons so you can always refere to `$scope.motionsFiltered`.